src/ui/templates/Welcome.jsx

```diff
@@ -59,39 +59,41 @@ function component({ version }) {
                     </p>
                 </div>
                 <div class="content_section">
-                    <h5>
-                        Caution! It is not recommended to downgrade the extension version to below
-                        2.0!
-                    </h5>
-                </div>
-                <div class="content_section">
-                    <h3>Exercise management</h3>
-                    <p>
-                        The main focus of this release is to introduce a new management system for
-                        exercises on disk. These features have been moved to TMC-langs that is the
-                        centralized utility for main TestMyCode related functions. This new
-                        architecture reduces previously existing major overheads and allows for some
-                        performance optimizations.
-                    </p>
+                    <h3>Exercise Decorations</h3>
                     <p>
-                        Unfortunately this change required for all exercises to be migrated to a new
-                        default location. If you would like to, you may now move them back again
-                        from the settings view.
+                        You can now see completed and partially completed (i.e. received some
+                        points) exercises with an icon on the course workspace.
+                        <br />
+                        We also implemented showing if the deadline has exceeded and if the exercise
+                        has been removed or renamed in the course. By hovering on an exercise with
+                        an icon, you should see an information message explaining the status.
                     </p>
+                    <img
+                        style="margin-bottom: 1em;"
+                        class="rounded mx-auto d-block"
+                        src={exerciseDecorations}
+                    />
                 </div>
                 <div class="content_section">
-                    <h3>Improved exercise downloads</h3>
+                    <h3>Migrated to VSCode Settings</h3>
                     <p>
-                        Following the change to exercise management, downloading new exercises is
-                        now significantly faster.
+                        We removed our own Settings webview and migrated the user and course
```

Comment:
Old custom settings view has been removed in favor of VS Code's native settings page.